File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L6/05/pengulangan.kt
package IR

fun hitHasil(a: Int) {
    for (i in a..10) {
        println("$i\t\t${i * 1.609}")
    }
}

fun main(args: Array<String>) {
    println("Miles\t\tKilometers")
    println("-------------------------------")
    hitHasil(1)
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L6/05/pengulangan.ts
package IR;

import java.util.*;

class inputoutput {
    static hitHasil(a: number): void {
        for (; a <= 10; a++) {
            console.log(`${a}\t\t${a * 1.609}`);
        }
    }

    public static main(args: string[]): void {
        // print seusatu
        console.log("Miles\t\tKilometers");
        console.log("-------------------------------");
        // pake pengulangan while
        inputoutput.hitHasil(1);
    }
}

